service/src/main/java/bio/terra/workspace/amalgam/landingzone/azure/LandingZoneApiDispatch.java

```diff
@@ -85,27 +66,25 @@ public ApiCreateLandingZoneResult createAzureLandingZone(
     // Prevent deploying more than 1 landing zone per billing profile
     verifyLandingZoneDoesNotExistForBillingProfile(bearerToken, body);
 
-    LandingZoneRequest landingZoneRequest =
-        LandingZoneRequest.builder()
-            .landingZoneId(body.getLandingZoneId())
-            .definition(body.getDefinition())
-            .version(body.getVersion())
-            .parameters(
-                MapperUtils.LandingZoneMapper.landingZoneParametersFrom(body.getParameters()))
-            .billingProfileId(body.getBillingProfileId())
-            .build();
-    return toApiCreateLandingZoneResult(
-        landingZoneService.startLandingZoneCreationJob(
-            bearerToken, body.getJobControl().getId(), landingZoneRequest, asyncResultEndpoint));
+    return amalgamated.startLandingZoneCreationJob(
```

Comment:
Why not start using the `WorkspaceLandingZoneService` interface type now?

It will always be `AmalgamatedLandingZoneService` until the other implementation is created and configured, but then this class shouldn't need further adaptation, unless the interface changes.

Reply:
Good call, I'll make the change.